runtime/runtime-params-estimator/src/vm_estimator.rs

```diff
@@ -162,6 +162,9 @@ pub fn cost_to_compile(
                     } else {
                         "wasmtime"
                     },
+                VMKind::Combined => {
+                    panic!("Combined VM is for execution only")
```

Comment:
Didn't see in this PR, how do we get combined runner's runtime params. Is that by add wasmer runtime params and wasmtime runtime params? If so, would be nice to add a short script to generate combined runtime params